pyccl/ccl_background.i.old

```diff
@@ -0,0 +1,150 @@
+%module ccl_background
```

Comment:
We shouldn't commit this file, github will keep track of the old version for us.

Reply:
slipped through. Removed!

Reply:
now removed.